wrapper/minecraft/src/main/kotlin/org/quiltmc/qkl/wrapper/minecraft/text/TextDsl.kt

```diff
@@ -193,7 +193,7 @@ public class TextDsl internal constructor(action: TextDsl.() -> Unit) {
      *
      * @author NoComment1105
      */
-    public fun getText(): Text {
+    internal fun getText(): Text {
```

Comment:
i'm not sure what's the point of this though, since the `text` property itself is public

Reply:
It was added when text was a private property, but we since made it public since inlining other functions would error otherwise

Reply:
right, so I suggest we just get rid of this method and use `.text` directly in `buildText`

Reply:
Ok will do